datasets/us/ma/med_exclusions/us_ma_med_exclusions.yaml

```diff
@@ -0,0 +1,34 @@
+title: US Massachusetts Medicaid Terminated Provider List
+entry_point: crawler.py
+prefix: us-medma
+coverage:
+  frequency: daily
+  start: "2024-08-15"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset contains a list of providers who have been terminated for cause from Massachusetts's Medicaid program.
+description: |
+  The MassHealth program maintains a list of providers who have been suspended or excluded
+  from participating in the MassHealth program. This list is updated monthly and reflects
+  suspensions or exclusions effective on or after March 23, 2010.
+publisher:
+  name: MassHealth
+  acronym: MASS
```

Comment:
I think MASS is short for massachusetts - this is the state gov website, not masshealth. I think MassHealth is sufficiently short that we can skip the acronym.

But let's add a publisher description

```suggestion
```